jar-service/pom.xml

```diff
@@ -200,8 +200,8 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-compiler-plugin</artifactId>
 				<configuration>
-					<target>1.6</target>
-					<source>1.6</source>
+					<target>1.7</target>
+					<source>1.7</source>
```

Comment:
#365 wasn't worked on yet.  After fixing it, the source/target versions should be defined in the parent pom as properties.